test/e2e/components/vl-select.js

```diff
@@ -27,14 +27,51 @@ class VlSelect extends VlElement {
     return container.hasClass('is-open');
   }
 
+  async groups() {
+    if (await this.isDressed()) {
```

Comment:
is het dan niet beter om te wachten tot de vl-select dressed is ipv. hier een else voor te voorzien?

Reply:
Het is niet "awaitUntil" dressed he. Het is gewoon het resultaat van ne  promise afwachten. 
Ge hebt gewoon een undressed/native variant van select en een dressed variant die wat fancy stuff doet ...

Reply:
Ja, mijn punt was vooral: waarom er geen awaitUntil van maken. Ik wist namelijk niet dat vl-select ook undressed te gebruiken is. Ik ging er vanuit dat hij zichzelf ook zou dressen, standaard.

Reply:
Nee dat kan je dus kiezen.